libraries/common/subscriptions/router.js

```diff
@@ -64,10 +69,30 @@ export default function router(subscribe) {
       }
     }
 
-    // Check for a redirect url and change location if one is found.
-    const redirect = redirects.get(location);
+    // Check for a redirect and change location if one is found.
+    let redirect = redirects.get(location);
 
     if (redirect) {
+      // Check if a redirect handler was assigned and execute it.
```

Comment:
This comment is not needed. It is obvious what's happening.

Reply:
done